src/bot.js

```diff
@@ -3,13 +3,26 @@ const config = require('./config/config');
 const utils = require('./utils');
 const logger = require('./logger');
 const db = require('./mongo');
+const fs = require('fs');
+
+
 
 let initializeBot = async() => {
 	const client = new Discord.Client({
 		disableEveryone: true,
 		autorun: true
 	});
+
+	client.commands = new Discord.Collection();
+	const commandFiles = fs.readdirSync('./src/commands').filter(file => file.endsWith('.js'));
+
+	for(const file of commandFiles) {
```

Comment:
Added a linting rule to fix this... xd. Pull latest and run lint when you have time.

Reply:
A rule for what exactly?

Reply:
Spaces before and after keywords. ‘for (...) {‘

Reply:
Spaces before and after keywords. ‘for (...) {‘